<launch>
  <node name="panda_cmd_client" pkg="rl_interface" type="cmd_client.py" ></node>
</launch>